src/app/components/configuration-modal/configuration-modal.component.html

```diff
@@ -4,7 +4,17 @@ <h1 mat-dialog-title>Board Configuration</h1>
     <mat-tab label="Board"> 
       <mat-form-field appearance="outline" style="width:100%; margin-top: 20px;">
         <mat-label>Board Name</mat-label>
-        <input matInput [(ngModel)]="boardName">
+        <input matInput [(ngModel)]="boardName" #boardNameInput="ngModel" required>
+        <mat-error *ngIf="boardNameInput.invalid">
+          Board Name is required.
+        </mat-error>
+      </mat-form-field>
+      <mat-form-field appearance="outline" style="width:100%; margin-top: 20px;">
+        <mat-label>Initial Zoom</mat-label>
+        <input matInput [(ngModel)]="initialZoom" #initialZoomInput="ngModel" type="number" min="1" required>
```

Comment:
Can we use a slider here instead of number input? I think it would be more user-friendly and we can probably find one from Material-UI.